pkg/api/metrics.go

```diff
@@ -16,13 +16,34 @@
 package api
 
 import (
+	"time"
+
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
 var (
 	MetricLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
 		Name: "timestamp_api_latency",
```

Comment:
Can we standardize the prefix on the metrics for all metrics? "tsa" or "ts" or "timestamp" or "timestamp_authority"?

Reply:
Just prefixed all the metrics with "timestamp_authority".